src/recipe/Baked.Recipe.Service.Application/Theme/Admin/Components.cs

```diff
@@ -45,32 +45,46 @@ public static ComponentDescriptorAttribute<DataTable> DataTable(
         int? rowsWhenLoading = default,
         string? scrollHeight = default,
         DataTable.Footer? footerTemplate = default,
+        DataTable.Export? exportOptions = default,
         IData? data = default
-    ) => new(
-        new()
+    )
+    {
+        if (exportOptions is not null)
         {
-            Columns = [.. columns ?? []],
-            DataKey = dataKey,
-            ItemsProp = itemsProp,
-            Paginator = paginator,
-            Rows = rows,
-            RowsWhenLoading = rowsWhenLoading,
-            ScrollHeight = scrollHeight,
-            FooterTemplate = footerTemplate
+            exportOptions.ButtonIcon ??= "pi pi-external-link";
```

Comment:
this icon is used in primevue samples but i think it's misleading. `pi-download` or `pi-file-export` suits more